blueprints/frontend-shared-config/files/.jscsrc

```diff
@@ -9,72 +9,32 @@
   "additionalRules": [ "lib/jscs-rules/*.js" ],
   "fileExtensions": [".js"],
   "esnext": true,
+  "requireCommaBeforeLineBreak": true,
+  "requireCurlyBraces": [
+    "if",
+    "else",
+    "for",
+    "while",
+    "do",
+    "try",
+    "catch"
+  ],
   "disallowEmptyBlocks": true,
-  "disallowMultipleSpaces": true,
+  "disallowIdentifierNames": ["proto", "_super"],
   "disallowKeywordsOnNewLine": ["else"],
   "disallowMixedSpacesAndTabs": true,
   "disallowMultipleLineBreaks": true,
-  "disallowMultipleLineStrings": true,
+  "disallowMultipleSpaces": true,
```

Comment:
Just a question about this rule, does it prevent variable declarations like:

```
var one              = 1;
var anotherOne = 1;
var someOne    = 1;
```

I usually don't use that style myself but wondering if that would apply here?